src/pydex_app/utils.py

```diff
@@ -88,3 +89,31 @@ def setup_logger(
     logger.setLevel(level)
 
     return logger
+
+
+def sign_order(order_hash, web3_instance, private_key):  # pylint: disable=redefined-outer-name
+    """Function to sign order_hash via eth-sign and convert ec_signature to
+    0x compatible signature
+    """
+
+    def to_32byte_hex(val):
+        """Method to convert value to bytes32 hex
+        """
+        return web3_instance.toHex(web3_instance.toBytes(val).rjust(32, b'\0'))
+
+    def convert_ec_sig(ec_signature):
+        """Make ec_signature compatible with 0x
```

Comment:
Would be great to expand the docstring here to explain what you are doing within this as it is non-trivial